docs/platform-support.md

```diff
@@ -0,0 +1,35 @@
+# Officially Supported Platforms
```

Comment:
Ah, one more thing, please also add the YAML preamble for the website, e.g. something like:

```
---
id: platform-support
title: Officially Supported Platforms
sidebar_label: Platform Support
---
```

Reply:
Done.